lib/Controller/ViewController.php

```diff
@@ -76,6 +86,7 @@ public function index():TemplateResponse {
 			'show_weekends' => $this->config->getUserValue($this->userId, $this->appName, 'showWeekends', 'yes') === 'yes',
 			'show_week_numbers' => $this->config->getUserValue($this->userId, $this->appName, 'showWeekNr', 'no') === 'yes',
 			'skip_popover' => $this->config->getUserValue($this->userId, $this->appName, 'skipPopover', 'no') === 'yes',
+			'talk_enabled' => $this->appManager->isEnabledForUser('spreed'),
```

Comment:
It really is time to itnroduce you to the initialstate api ;) @ChristophWurst we must spread the word of it!

Reply:
Yes, it's on my never ending todo list ;)